backend/src/common/client/api_server/v2/experiment_client.go

```diff
@@ -0,0 +1,217 @@
+// Copyright 2018-2023 The Kubeflow Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package api_server_v2
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/go-openapi/runtime"
+	"github.com/go-openapi/strfmt"
+	apiclient "github.com/kubeflow/pipelines/backend/api/v2beta1/go_http_client/experiment_client"
+	params "github.com/kubeflow/pipelines/backend/api/v2beta1/go_http_client/experiment_client/experiment_service"
+	model "github.com/kubeflow/pipelines/backend/api/v2beta1/go_http_client/experiment_model"
+	"github.com/kubeflow/pipelines/backend/src/common/client/api_server"
```

Comment:
Would it be better to move v1 clients into `common/client/v1` and v2 clients into `/common/client/v2`, while keeping the shared utils in `/common/client`?

Reply:
As we are going to support both v1 and v2 API once integration is completed, should we make a single package for both API clients? If yes, we may want to store v2-related code in the same folder with the v1?

Reply:
> Would it be better to move v1 clients into common/client/v1 and v2 clients into /common/client/v2, while keeping the shared utils in /common/client?

Yes, I can do that, it would just produce more diff with not very obvious benefits IMO. But I don't have a strong opinion. I can make the change.

> As we are going to support both v1 and v2 API once integration is completed, should we make a single package for both API clients? If yes, we may want to store v2-related code in the same folder with the v1?

We will support both v1 and v2 API in the foreseeable near term, and deprecate v1 API later. And the clients are only used for testing only, so I don't think it's worth the effort to combine them into a single package.

Reply:
Separating clients (even only experiment client) to v1 and v2 folder creates diffs for the entire integration test suite. I'll keep this PR clean to initialization test at this moment, and rethink whether to move v1 clients when adding v2 integration tests.